ton/wallet/wallet.go

```diff
@@ -465,6 +465,32 @@ func (w *Wallet) SendWaitTransaction(ctx context.Context, message *Message) (*tl
 	return w.SendManyWaitTransaction(ctx, []*Message{message})
 }
 
+// BroadcastTransactionsAndWait broadcasts the transaction and waits for confirmation, returning the transaction, block, and any errors.
+func (w *Wallet) BroadcastTransactionsAndWait(ctx context.Context, addr *address.Address, amount tlb.Coins, comment string) (*tlb.Transaction, *ton.BlockIDExt, error) {
+	var body *cell.Cell
+	var err error
+
+	if comment != "" {
```

Comment:
Lets reuse w.BuildTransfer method here, it is doing exactly same. 

For the bounce flag we can take it from `addr.IsBounceable()`, so it will be more predictable